.github/workflows/pdm-lock-automation.yaml

```
@@ -66,30 +71,15 @@ jobs:
           fi
           source .venv/bin/activate
           pdm lock -G :all
```

```suggestion
          pdm lock -G :all -v
```
Making it verbose to quickly analyze any issues

---

done @chandrasekharan-zipstack